parser_test.go

```diff
@@ -346,6 +346,26 @@ func TestParseRule(t *testing.T) {
 				Tags: map[string]string{"flow": "to_server,established", "classtype": "trojan-activity"},
 			},
 		},
+		{
+			name: "Metadata",
```

Comment:
optional: you can simplify this test a bit by removing some of the unneeded pieces (contents, etc)